### PR TITLE
fix(typos): Fix typos thanks to the Vale tool

### PR DESCRIPTION
**Pull request by jeromecambon**
This is a first set of typos fix, found with the Vale tool.

See also #2195

### CODE REVIEW

File: modules/ROOT/pages/engine-architecture-overview.adoc [hunk 1]
```diff
@@ -334,7 +334,7 @@ Generic services  perform actions that are not related to BPM but are required f
 | Uses the Identity service to check user credentials
 |===
 
-=== Queriable logs
+=== Queryable logs
```
Comment (tbouffard) [inline modules/ROOT/pages/engine-architecture-overview.adoc]: ⚠️ not sure we can change this, even if incorrect if this is the name that had been choosen in the code 😞

File: modules/ROOT/pages/engine-architecture-overview.adoc [hunk 2]
```diff
@@ -703,7 +703,7 @@ Generic services  perform actions that are not related to BPM but are required f
 |  |
 
 | Description:
-| A layer between the persistence service and all services that need to store objects. It ensures that events are triggered and that queriable log entries are written.
+| A layer between the persistence service and all services that need to store objects. It ensures that events are triggered and that queryable log entries are written.
```
Comment (tbouffard) [inline modules/ROOT/pages/engine-architecture-overview.adoc]: ⚠️ same remark as above

File: modules/ROOT/pages/set-log-and-archive-levels.adoc [hunk 3]
```diff
@@ -9,7 +9,7 @@ There are three ways to record the information of a process:
 
 * Archive: used by a business analyst or process administrator using Bonita Portal, during the production phase, to trace the running of the process instances in their life cycle. Records who did what and when.
 * Technical log: used by an application developer for troubleshooting, debugging and process tuning.
-* Queriable log: used by a database administrator extracting information related to business actions that is stored in the internal database to be retrieved by a database request.
+* Queryable log: used by a database administrator extracting information related to business actions that is stored in the internal database to be retrieved by a database request.
```
Comment (tbouffard) [inline modules/ROOT/pages/set-log-and-archive-levels.adoc]: ⚠️ same remark as in `modules/ROOT/pages/engine-architecture-overview.adoc`

File: modules/ROOT/pages/salesforce.adoc [hunk 4]
```diff
@@ -1,7 +1,7 @@
-= Using SaleForce with Bonita
-:description: SaleForce connectors allow you to interact with Saleforce through Bonita.
+= Using Salesorce with Bonita
```
Comment (tbouffard) [inline modules/ROOT/pages/salesforce.adoc]:
```suggestion
= Using Salesforce with Bonita
```

**Review by tbouffard — CHANGES_REQUESTED** [review]
Most of the changes are perfect
However, I suggest we postpone changes about "Queriable log" / "Queryable log" (see comments)

**Comment by tbouffard** [comment]
❓ @jeromecambon can you explain which vale command you ran? I guess it is also using the valve configuration provided in #2195

**Comment by jeromecambon** [comment]
> ❓ @jeromecambon can you explain which vale command you ran? I guess it is also using the valve configuration provided in #2195

I ran :
`vale --minAlertLevel=error --output line .`
and yes, I used the Vale config you mentioned.

**Review by tbouffard — APPROVED** [review]
LGTM the ~~valve~~ **Vale** tool is really promising!

**Comment by jeromecambon** [comment]
> LGTM the valve tool is really promising!

Yes, and its name is **Vale** ;-)

**Pull request merged**